ppl/archive/seekindex/seekindex_test.go

```diff
@@ -0,0 +1,103 @@
+package seekindex
+
+import (
+	"context"
+	"io/ioutil"
+	"math"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/brimsec/zq/pkg/iosrc"
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/zbuf"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAscending(t *testing.T) {
+	var entries = []entry{
+		{1490469784771872000, 0},
```

Comment:
Would you change the timestamps here so that they're easier to read at a glance, as in TestDescending?